go/enclave/evm/evm_facade.go

```diff
@@ -138,6 +181,18 @@ func (exec *evmExecutor) ExecuteCall(ctx context.Context, msg *gethcore.Message,
 		NoBaseFee: true,
 	}
 
+	// a call can create multiple contracts; capture via tracer
+	var createdContracts []*gethcommon.Address
```

Comment:
can we pass in a parameter to ExecuteCall, so we know whether it's an estimate gas?
My concern is that this hook will slow things down for all calls, and it's only needed for estimate_gas